Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG ruby_version=3.3
+ARG ruby_version=3.3.1
```

Comment:
Please don't pin patch versions in the Dockerfile, otherwise you'll prevent security fixes from happening.

Reply:
Sure. I've done a quick GitHub search and saw the Dockerfile being change this way so followed suit.

Reply:
Eek, there are indeed a handful of Dockerfiles (5/49) with obsolete pinned versions. Best not to copy without understanding. Thanks for pointing it out — I can go fix those now.

```
~/s/alphagov> rg -gDockerfile 'ruby_version.\d+\.\d+\.'
specialist-publisher/Dockerfile:1:1:ARG ruby_version=3.2.4
publisher/Dockerfile:1:1:ARG ruby_version=3.3.1
search-admin/Dockerfile:1:1:ARG ruby_version=3.3.1
publishing-api/Dockerfile:1:1:ARG ruby_version=3.3.1
search-api-v2/Dockerfile:1:1:ARG ruby_version=3.3.1
```